### DISCUSSION TITLE
I'm using Windows 10, can I make a production build for both MAC and Windows?

### DISCUSSION THREAD

**Discussion by JenuelDev** [nklayman/vue-cli-plugin-electron-builder#1521] [Q&A] [answered]
I'm using Windows 10, can I make a production build for both MAC and Windows? I heard you can do it using electron forge, but I don't know how to do it. Thanks for any answers :)

**Comment by nklayman** [comment, marked as answer]
I don't think you can make a Mac build from Windows, and even if you could you definitely can't sign it, which is pretty much a requirement for deploying any mac app. See the [Electron Builder docs](https://www.electron.build/multi-platform-build.html) for more information.

**Reply by JenuelDev** [reply]
what if I use virtual machine? is it possible?

**Reply by klren0312** [reply]
> what if I use virtual machine? is it possible?

That's definitely ok
